### PR TITLE
[4.0] Toggle Password Visibility - Installer

### PR DESCRIPTION
**Pull request by brianteeman**
The strings for Screen Readers are now "Show Password" and "Hide Password"

NOTE: Before these recent PR the Text was untranslated

### CODE REVIEW

File: installation/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -221,6 +221,7 @@ JADMINISTRATOR="Administrator"
 JERROR="Error"
 JERROR_LAYOUT_ERROR_HAS_OCCURRED_WHILE_PROCESSING_YOUR_REQUEST="An error has occurred while processing your request."
 JEMAIL="Email"
+JHIDEPASSWORD="Hide Password"
```
Comment (Quy) [inline installation/language/en-GB/joomla.ini]: Alpha order

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fdd641e33110b63b79d8707ab7cedd7386ac12a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27428">issues.joomla.org/tracker/joomla-cms/27428</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on fdd641e33110b63b79d8707ab7cedd7386ac12a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27428">issues.joomla.org/tracker/joomla-cms/27428</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27428">issues.joomla.org/tracker/joomla-cms/27428</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**